code-of-conduct.md

```diff
@@ -1,38 +1,134 @@
-environment where anyone can participate and everyone can make a difference.
-
-## Considerate
-Members of the community are considerate of their peers -- other Python users.
-We're thoughtful when addressing the efforts of others, keeping in mind that
-often times the labor was completed simply for the good of the community. We're
-attentive in our communications, whether in person or online, and we're tactful
-when approaching differing views.
-
-## Respectful
-Members of the community are respectful. We're respectful of others, their
-positions, their skills, their commitments, and their efforts. We're respectful
-of the volunteer efforts that permeate the Python community. We're respectful
-of the processes set forth in the community, and we work within them. When we
-disagree, we are courteous in raising our issues.
-
-Overall, we're good to each other. We contribute to this community not because
-we have to, but because we want to. If we remember that, these guidelines will
-come naturally.
-
-For any issues under this code, or questions about it, please contact the ChiPy
-organizers at (chicago-organizers@python.org) or approach any of the organizers
-directly in person or on Slack.
+# Code of Conduct
+
+The Chicago Python User Group(ChiPy) community is made up of active members
+from the greater Chicago-land areas and participating virtually with a
+diverse set of skills, personalities, and experiences. It is through this
+diversity that our community experiences great successes and continued
+growth.
+
+We value the participation of each member of the Python community and want all
+participants to have an enjoyable and fulfilling experience. Accordingly, all
+participants are expected to show respect and courtesy to other participants at
+all ChiPy events.
+
+To make clear what is expected, all participants, speakers, sponsors,
+organizers, and volunteers at any ChiPy event are required to conform to the
+following Code of Conduct. Organizers will enforce this code at every event.
+
+## Our Standards
+
+ChiPy is dedicated to providing a positive experience for everyone, regardless
+of age, gender identity and expression, sexual orientation, disability,
+physical appearance, body size, ethnicity, nationality, citizenship, color,
+race, or religion (or lack thereof), pregnancy, education, or socio-economic
+status.
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+- Being kind to others
+- Behaving professionally
+- Using welcoming and inclusive language
+- Being respectful of differing viewpoints and experiences
+- Gracefully accepting constructive criticism
+- Focusing on what is best for the community
+- Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+- Continued one-on-one communication after requests to cease
+- Deliberate misgendering, using inappropriate pronouns, or use of “dead” or
```

Comment:
Can we clarify what "dead" names means? I'm hazy on that.

Reply:
+1 - adding a reference would be very helpful.

Reply:
I forgot to mention, I use Party Hat to mean "Yay and I did it!" This comment thread is 🚢.